app/views/static/home.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="container">
+  <div class="pages-header">
+  </div>
+  <div class="pages-header">
+    <span class="center">Start taking online payments!</span>
+      <%= link_to "Landlord Sign Up", new_landlord_path, class: 'center btn btn-secondary' %>
+    </div>
+  </div>
```

Comment:
The reason the first line is indented is because of the closing </div> on line 8. This div closes the div started on line 1 with a class of `container`.  The indentation is off making that hard to see.  It looks like this extra div is in each of the page header sections. Remove lines 8, 12, 17 and 22 and it should look correctly.